_includes/donation_banner.html

```diff
@@ -37,7 +37,7 @@
         </p>
       </div>
       <button class="donate-button" id="donate-button"><a
-        href="https://www.paypal.com/donate?campaign_id=NEAV3YL4H6B5S">DONATE NOW</a></button><div class="donate-close-button">
+        href="https://www.every.org/sugar-labs?amount=17&frequency=ONCE#donate">DONATE NOW</a></button><div class="donate-close-button">
```

Comment:
I think we want this instead: https://www.every.org/sugar-labs?suggestedAmounts=37%2C97%2C177&min_value=17&utm_campaign=donate-link#/donate